Project/MyGameLibrary/Character/Character.cs

```diff
@@ -1,29 +1,32 @@
 ﻿using System.Collections.Generic;
+using MyGameLibrary.Shop;
 
-namespace Fall2020_CSC403_Project.code
+namespace MyGameLibrary.Character
 {
     public class Character
     {
-        public string CharacterName { get; set; };
-        public bool isDead = false;
+        public string CharacterName { get; set; }
+        public bool IsDead { get; set; }
         private int LoveScore;
         public int GetLoveScore() {
```

Comment:
Please use C#'s method of getting and setting, see previous comment as an example